src/nls/root/shortcuts-win.js

```diff
@@ -38,6 +38,7 @@ define(function (require, exports, module) {
                 SAVE: "s",
                 SAVE_AS: "s",
                 REVERT: "KEY_F12",
+                EXPORT_UTILITY: "e",
```

Comment:
Seth suggested Cmd/Ctrl+e, and I couldn't find any conflict with existing Ps shortcuts.